docs/harvester-network.md

```diff
@@ -0,0 +1,70 @@
+## Introduction of The Harvester network
+
+### Table of Contents
+* [Introduction of The Harvester network](#introduction-of-the-harvester-network)
+  * [Summary](#summary)
+  * [Implementation](#implementation)
+    * [Management Network](#management-network)
+    * [VLAN](#vlan)
+  * [UI Interaction](#ui-interaction)
+
+
+### Summary
+[Harvester](https://github.com/rancher/harvester) is built on Kubernetes, which uses [CNI](https://github.com/containernetworking/cni) as an interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, [Harvester UI](https://github.com/rancher/harvester-ui) integrates the Harvester network to provide a user-friendly way to configure networks for VMs.
+
+By version 0.2, Harvester supports two kinds of networks: 
+- management network
+- [VLAN](https://en.wikipedia.org/wiki/Virtual_LAN)
+
+### Implementation
+
+#### Management Network
+
+Harvester adopts [flannel](https://github.com/flannel-io/flannel) as the default CNI to implement the management network. It's an internal network, which means the user can only access the VM's management network within its cluster nodes or pods.
+
+#### VLAN
+
+[Harvester network-controller](https://github.com/rancher/harvester-network-controller) leverages the [multus](https://github.com/k8snetworkplumbingwg/multus-cni) and [bridge](https://www.cni.dev/plugins/current/main/bridge/) CNI plugins to implement the VLAN.  
+
+Below is a use case of the VLAN in Harvester.
+
+![image-20210412181210146](https://raw.githubusercontent.com/yaocw2020/img/master/image-20210412181210146.png)
```

Comment:
Can we put all image files to https://github.com/rancher/harvester/tree/master/docs/assets